index.html

```diff
@@ -27,6 +27,39 @@
     <div id="text"></div>
     <div><br /><button id="restart">RESTART</button></div>
     <footer>
+      <br />
+      <form>
+        <label id="P1Name">Player X:</label>
```

Comment:
I don't think you need to give the label an `id`
I'm not sure you need to target it for anything
but there is a `for` property that should match the `id` value of the form element it is a label for

example: https://www.w3schools.com/tags/tag_label.asp